apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -153,7 +155,7 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
         addUnhashedIdentifier(
```

Comment:
`MbiEffectiveDate` and `MbiObsoleteDate` are not getting set for the historic identifiers, resulting in no `period` being set and making them look current:

![image](https://user-images.githubusercontent.com/29063436/115896931-8c521e00-a421-11eb-8a23-ac6cd187b394.png)

Reply:
Yup, some of this is a function of Samle A data and code from stu3. Per our slack chat, I will not process MedicareBeneficiaryIdHistory rcds if they don't have an end date. This is because the MedicareBeneficiaryIdHistory table includes the current MBI id even though that is also in the corresponding Beneficiaries table rcd.

Also per our discussion, putting an MBI Extension Idnentifier that has no end date might have recipients assuming that it is the CURRENT MBI id.